aten/src/ATen/native/Embedding.cpp

```diff
@@ -154,13 +152,13 @@ Tensor & embedding_renorm_cpu_(
     Tensor & self, const Tensor & indices, double max_norm, double norm_type) {
   auto self_arg = TensorArg(self, "self", 1);
   auto indices_arg = TensorArg(indices, "indices", 2);
-  checkContiguous("embedding_renorm_", self_arg);
```

Comment:
You dropped this test but I don't see a subsequent contiguous() call. Does slicing and then norming work, no matter the stride? I guess there is no reason it shouldn't.

Reply:
Yeah, since it's a slicing on the tensor, it should just work.